storage/migrations/31_runtime_txs_oasis_encryption.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+ALTER TABLE chain.runtime_transactions
+    -- Encrypted data in encrypted Oasis-format transactions.
+    ADD COLUMN oasis_encrypted_format call_format,
+    ADD COLUMN oasis_encrypted_public_key BYTEA,
+    ADD COLUMN oasis_encrypted_data_nonce BYTEA,
+    ADD COLUMN oasis_encrypted_data_data BYTEA,
+    ADD COLUMN oasis_encrypted_result_nonce BYTEA,
+    ADD COLUMN oasis_encrypted_result_data BYTEA;
```

Comment:
Would it make sense to rename the existing `evm_encrypted_*` fields to drop the `evm_` prefix and reuse those fields for the encrypted tx data here? And then have some flag to indicate whether the encrypted tx was on the sdk level or in evm?

Reply:
what's the benefit, narrower table? dunno if that's worth the effort to deal with the extra logic to combine them when inserting and separate them when selecting

Reply:
Makes sense, let's go with the current form then 👍